.github/workflows/element-with-modules.yml

```diff
@@ -0,0 +1,351 @@
+          echo "  - '$VERSION'" >> ../build_config.yaml
+          HUMAN_TOGGLES="+wt-${HUMAN_TOGGLES}"
+          echo "HUMAN_TOGGLES=$HUMAN_TOGGLES" >> $GITHUB_ENV
+
+      - name: Checkout Dockerfile version
+        uses: actions/checkout@v4
+
+      - name: Checkout Element version
+        uses: actions/checkout@v4
+        with:
+          repository: element-hq/element-web
+          ref: ${{ matrix.element }}
+          path: element-web
+          fetch-tags: true
+
+      - name: Determine human readable Element Web version
+        env:
+          IS_RELEASE: ${{ matrix.element == 'staging' || matrix.element == 'master' || startsWith(matrix.element,'v') }}
+        working-directory: element-web
+        run: |
+          HUMAN_ELEMENT="$(git describe --exclude='*' --always)"
+          if ${{ env.IS_RELEASE }}; then HUMAN_ELEMENT="$(git describe --abbrev=0)"; fi
+          echo "HUMAN_ELEMENT=$HUMAN_ELEMENT" >> $GITHUB_ENV
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to ghcr.io
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Generate Docker metadata
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ghcr.io/nordeck/element-web-modules
+          labels: |
+            org.opencontainers.image.title=Element Web with modules
+            org.opencontainers.image.description=A ready to use Element Web with all Nordeck modules.
+            org.opencontainers.image.vendor=Nordeck IT + Consulting GmbH
+          tags: |
+            type=raw,value=ew-${{ env.HUMAN_ELEMENT }}${{ env.HUMAN_GUEST }}${{ env.HUMAN_OPENDESK }}${{ env.HUMAN_LIFECYCLE }}${{ env.HUMAN_TOGGLES }}
+          # It can't start with a period or hyphen and must be no longer than 128 characters.
+
+      - name: Collect module source files
+        run: |
+          mv ../build_config.yaml e2e/src/deploy/elementWeb/
+          mv ../customisations.json e2e/src/deploy/elementWeb/ || ${{ matrix.guest == 'none' }}
+          mv ../nordeck-element-web-guest-module.tgz e2e/src/deploy/elementWeb/ || ${{ startsWith(matrix.guest, '@nordeck') || startsWith(matrix.guest, 'none') }}
+          mv ../nordeck-element-web-opendesk-module.tgz e2e/src/deploy/elementWeb/ || ${{ startsWith(matrix.opendesk, '@nordeck') || startsWith(matrix.opendesk, 'none') }}
+          mv ../nordeck-element-web-widget-lifecycle-module.tgz e2e/src/deploy/elementWeb/ || ${{ startsWith(matrix.lifecycle, '@nordeck') || startsWith(matrix.lifecycle, 'none') }}
+          mv ../nordeck-element-web-widget-toggles-module.tgz e2e/src/deploy/elementWeb/ || ${{ startsWith(matrix.toggles, '@nordeck') || startsWith(matrix.toggles, 'none') }}
+
+      - name: Build
+        uses: docker/build-push-action@v6
+        with:
+          context: e2e/src/deploy/elementWeb/
```

Comment:
We should add something to the e2e README, that this is now also used to build and provide an image. It currently only mentions, that it is for testing.